src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DayRent = 40;
+  let totalRent = 0;
+
+  totalRent = DayRent * days;
+
+  if (days >= 7) {
+    totalRent = DayRent * days - 50;
+  }
+
+  if (days >= 3 && days <= 6) {
+    totalRent = DayRent * days - 20;
```

Comment:
It is not necessary to check if `days <= 6` since this condition will naturally follow if the first `if` condition is not met. Also, use a constant for the `3` and `20` similar to the above comment.